crates/holochain_p2p/src/spawn/actor.rs

```diff
@@ -805,6 +952,17 @@ impl actor::HcP2p for HolochainP2pActor {
         })
     }
 
+    fn peer_store(&self, dna_hash: DnaHash) -> BoxFut<'_, HolochainP2pResult<DynPeerStore>> {
+        Box::pin(async move {
+            Ok(self
+                .kitsune
+                .space(dna_hash.to_k2_space())
+                .await?
+                .peer_store()
+                .clone())
+        })
+    }
```

Comment:
Adding this api for testing, but I think sweettest is going to need it too, right?

Reply:
put a `#[cfg(test)]` on it?

Reply:
Cool, okay